lib/widgets/loading_indicator.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
```

Comment:
suggestion:  I'm not sure we need a widget just to wrap a progress indicator in a center.  Since this is only used in one place, I'd suggest removing it.  Just a thought.